packages/core/src/collision/b2_broad_phase.ts

```diff
@@ -215,8 +215,11 @@ export class b2BroadPhase<T> {
         ++this.m_moveCount;
     }
 
-    private UnBufferMove(proxy: b2TreeNode<T>): void {
-        const i = this.m_moveBuffer.indexOf(proxy);
-        this.m_moveBuffer[i] = null;
-    }
+    public UnBufferMove(proxy: b2TreeNode<T>): void {
```

Comment:
Why was this changed to public?

Reply:
Oh, I didn’t realize this had changed. I’ve fixed it now.